centralstore/redis_store.go

```diff
@@ -183,6 +183,18 @@ func (r *RedisBasicStore) WriteSpans(ctx context.Context, spans []*CentralSpan)
 	conn := r.RedisClient.Get()
 	defer conn.Close()
 
+	traceIDs := make([]string, 0, len(spans))
+	for _, span := range spans {
+		traceIDs = append(traceIDs, span.TraceID)
```

Comment:
Is it possible for 2 spans in the span list to have the same TraceID? Should we use a map/set instead of a list to ensure traceid is unique?

Reply:
Yes! Good catch! I will make the change